src/schema/v2/homeView/sections/DiscoverSomethingNew.ts

```diff
@@ -0,0 +1,46 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+import { connectionFromArray } from "graphql-relay"
+
+const marketingCollectionSlugs = [
+  "figurative-art",
+  "new-from-leading-galleries",
+  "paintings",
+  "prints",
+  "street-art",
+  "black-and-white",
+  "art-under-1000",
+  "art-for-small-spaces",
+  "cool-toned-artworks",
+]
+
+export const DiscoverSomethingNew: HomeViewSection = {
+  id: "home-view-section-discover-something-new",
+  featureFlag: "diamond_home-view-marketing-collection-categories",
+  contextModule: "" as ContextModule, // TODO: fill in with real value
+  type: HomeViewSectionTypeNames.HomeViewSectionCards,
+  component: {
+    title: "Discover Something New",
+  },
+  requiresAuthentication: false,
+  resolver: async (_parent, args, context, _info) => {
+    const { body } = await context.marketingCollectionsLoader({
+      slugs: marketingCollectionSlugs,
+    })
```

Comment:
**walkthrough**: Since marketing collections are modeled in a first class way in our backend, I experimented with using a data loader to fetch collection details. This should be easily cacheable as the list of slugs is hardcoded above.